src/App.jsx

```diff
@@ -9,7 +10,8 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article {...article} />
```

Comment:
it'd be better to pass values explicitly, like `<Component prop={value} prop2={value2} />